nss-day-cohort-30/bangazon-api-whimsical-whooping-cranes
# Specific customer

# GetCustomers query returns list of customers, and related products. Specific customer query doesn't have that additional information included yet. That will be another branch.

## Type of change
Please delete options that are not relevant.
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout specificCustomer`
3.git pull origin specificCustomer
3. dotnet start BangazonAPI
4. run get all customers in Postman by using localhost and" /api/customers" and then 
 localhost:port# and "/api/customers?_include=products to see customers wiht products
5. run test "Test_Get_All_Customers" in TestBangazonAPI/TestCustomers.

# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] My changes generate no new warnings
- [ x] I have added test instructions that prove my fix is effective or that my feature works
link to issue ticket:
https://github.com/nss-day-cohort-30/bangazon-api-whimsical-whooping-cranes/issues/2
